### ISSUE TITLE
Ernie 3.0 Progressive Pretraining Schedules

### ISSUE THREAD

**Issue by sshleifer** [PaddlePaddle/ERNIE#724]
Loved the Ernie 3.0 paper!
 
I have a question about the “progressive pretraining” settings:
 
In section 3.3.1 on page 6, it says:
 
> We propose to progressively, smoothly, simultaneously increase: input sequence length, batch size, learning rate, dropout rate.
 
In section 5, how do the smaller models get to the same loss as the large model in only 10 hours on 8 v100s?
 
> As shown in Tab. 11, we record the time for the loss value of the model converges to the same as that of the ERNIE 3.0.
 
In section 5, it seems that only batch size and sequence length are scheduled. And “dropout keeps 0 in the progressive warmup stage”. Is this correct?
 
 
To summarize, my 2 questions are:
1) Which variables should be increased during training?
2) How many steps (and with what schedules) are the models in section 5 trained for?
 
 
Thanks in advance and let me know if I should ask somewhere else!

**Comment by sshleifer** [comment]
anybody know the answer?

**Comment by XiaoqingNLP** [comment]
Have you solved your question? I have the same question @sshleifer
